ImmersiveEquipmentMeshGen/MeshHandler.cs

```diff
@@ -300,6 +306,132 @@ public void GenerateMesh(NifFile nif, string originalPath, string newPath, Model
             }
         }
 
+        /* engine reverts to defaults if it can't read files (the archaic GetPrivateProfile api is used), we might as well 
+         * throw and force the user to sort out their stuff */
```

Comment:
comment out of sync with code?

Reply:
It's refering to OpenRead which throws if the file can't be read for any reason